File: API_MOVIL/API_MOVIL/Models/IncotermType.cs
using System;
using System.Collections.Generic;

namespace API_MOVIL.Models;

public partial class IncotermType
{
    public int Id { get; set; }

    public string? Code { get; set; }

    public string? Name { get; set; }

    public virtual ICollection<Incoterm> Incoterms { get; set; } = new List<Incoterm>();
}

File: SIMEXWeb/backend/app/Models/IncotermType.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

/**
 * Tipo de Incoterm estandarizado (EXW, FOB, CIF, DAP…).
 *
 * Agrupa los pasos concretos (`Incoterm`) que componen cada tipo.
 *
 * @property int    $id
 * @property string $code Código ISO del Incoterm.
 * @property string $name Nombre legible.
 */
class IncotermType extends Model
{
    use HasFactory;

    public $timestamps = false;

    protected $fillable = ['code', 'name'];

    /**
     * Pasos concretos que componen este tipo Incoterm.
     *
     * @return HasMany
     */
    public function incoterms(): HasMany
    {
        return $this->hasMany(Incoterm::class);
    }
}
